photo_points/ViewModels/PhotoStreamViewModel.cs

```diff
@@ -0,0 +1,11 @@
+﻿using photo_points.Models;
```

Comment:
Since you are not referencing ImageSource in any code, having this extra class is unnecessary. Where you are using this class, you can substitute List<Capture>

Reply:
Thanks! I'll remove that.